src/Validator/Constraints/CalculationModificationMemberValidator.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Validator\Constraints;
+
+use App\Entity\CalculationModification;
+use App\Entity\Member;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class CalculationModificationMemberValidator extends ConstraintValidator
+{
+
+    public function validate($value, Constraint $constraint): void
+    {
+        /** @var CalculationModification $object */
+        $object = $this->context->getObject();
+        $calculationModificationEvent = $object->getPayment()->getEvent()->getUid();
+        /** @var Member $member */
+        foreach ($value->getValues() as $member) {
```

Comment:
### 🤔 Question: Can `$value->getValues()` return an empty array?

If it can - u should check it at the top of func

Reply:
value->getValues() cannot be empty